.github/scripts/parse_version.bash

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
```

Comment:
I think it would be worth describing what this does so others who might have to maintain (hopefully never) would know the purpose

Reply:
HA! It should be `/bin/bash`. I'm going to rename the script.